.env.development

```diff
@@ -14,3 +14,4 @@ REACT_APP_ENABLE_MESSAGING=true
 REACT_APP_ENABLE_REGIONS=true
 REACT_APP_ENABLE_TOOLTIPS=true
 REACT_APP_ENABLE_STAKEHOLDERS=true
+REACT_APP_ENABLE_GENDER_REPORTING_CARD=true
```

Comment:
The API is already deployed to production. I don't think we need a feature flag for this.